app/views/videos/show.html.haml

```diff
@@ -8,41 +8,31 @@
           %h3 #{@video.title}
           %span Rating: 4.5/5.0
         %p #{@video.description}
-        .actions
-          %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
+
 
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form
+      = form_for [@video, Review.new] do |f|
         %fieldset
           .form-group
             %label Rate this video
-            .row
-              .col-sm-3
-                %select.form-control(name="")
-                  %option(value="5") 5 Stars
-                  %option(value="4") 4 Stars
-                  %option(value="3") 3 Stars
-                  %option(value="2") 2 Stars
-                  %option(value="1") 1 Star
-          .form-group
+            = f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number,"star")]})
```

Comment:
I would suggest to move this to a helper method `options_for_select([5,4,3,2,1].map {|number| [pluralize(number,"star")]})`.